pkg/reconciler/kubernetes/tektonresult/transform.go

```diff
@@ -82,6 +84,18 @@ func (r *Reconciler) transform(ctx context.Context, manifest *mf.Manifest, comp
 	return common.Transform(ctx, manifest, instance, extra...)
 }
 
+func filterExternalDB(p v1alpha1.ResultsAPIProperties) mf.Transformer {
```

Comment:
i dont know, what this is doing

Reply:
@khrm I too have same question. Could you please elaborate about this transformer?

Reply:
It removes the statefulset for DB.

Reply:
if you want to remove this resource from manifest, you should do a filterning for that resource based on flag,

not transform like this

Reply:
Filter doesn't remove resources. It uses a deepcopy. 
As manifestival write: 
```
The only way to alter resources in a Manifest is with its Transform method.
```

Reply:
Changed to using a filter.

Reply:
Putting this to resolved.